checker/jtreg/nullness/onlyannotatedfor/AnnotatedForWithUse.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * @test
+ *
+ * @summary Test that difference default applies to unannotated code.
+ * @compile/fail/ref=OnlyAnnotatedForWithUse.out -XDrawDiagnostics -Xlint:unchecked -processor org.checkerframework.checker.nullness.NullnessChecker -AonlyAnnotatedFor AnnotatedForWithUse.java
+ * @compile/fail/ref=ConservativeDefaultWithUse.out -XDrawDiagnostics -Xlint:unchecked -processor org.checkerframework.checker.nullness.NullnessChecker -AuseConservativeDefaultsForUncheckedCode=source AnnotatedForWithUse.java
```

Comment:
Should there be a third execution (first or last?) that executes with the default flags?